website/src/theme/Footer/styles.module.css

```
@@ -0,0 +1,323 @@
+}
+
+/*
+ * One glyph rule, one mask variable — an icon is a class that sets the variable,
+ * not another block of CSS. `currentColor` means the hover above recolours the
+ * glyph for free.
+ */
+.glyph {
+  flex: none;
+  width: 1.1rem;
+  height: 1.1rem;
+  background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)